packages/nimble-components/src/listbox/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+
+import {
+    applicationBackgroundColor,
+    borderWidth,
+    popupBorderColor,
+    elevation2BoxShadow,
+    bodyFont,
+    bodyFontColor,
+    smallPadding,
+    menuMinWidth
+} from '../theme-provider/design-tokens';
+
+export const styles = css`
```

Comment:
How should sizing behave for this component? It looks like it currently attempts to grow its height to fit the number of options and grow its width to fit the length of the longest option. If you constrain it with a small `max-height` it renders options outside the bounds of the listbox and with a `max-width` it wraps option text.

I think the height behavior is OK if we state / assume that we expect clients to put the listbox in a scrolllable parent if needed.

The text wrapping seems a bit weirder to me since it's very possible that user names will be longer than what we design for and I don't think fixed height components should wrap, they should ellipsize text. Thoughts on adding styles to ellipsize? 

If we do then we should add matrix test coverage for that case.

Reply:
Yeah, we could have ellipsis text especially when there is a long user name on a small mobile screen, instead of hiding the popup we could show the ellipsis text. To achieve this, I have updated the content element in `nimble-list-option` that is given by a client to this component.

Reply:
Thanks for looking into it. I hadn't considered that it would be a list-option change, which means it impacts other components like select and combobox too. We should probably add test coverage to those components' matrix tests for this case. It also brings up an accessibility question: how does the user access the full text? Do we need to add a tooltip?

Given this complexity I would actually vote to do this in a separate PR or file an issue to track it. This PR can keep the previous behavior of wrapping which is not great but an OK behavior while we're still developing.

Reply:
Created an issue to track the work: https://github.com/ni/nimble/issues/1658